src/rpc/common/impl/HandlerProvider.hpp

```diff
@@ -40,7 +39,7 @@ namespace rpc {
 class Counters;
 }  // namespace rpc
 namespace feed {
-class SubscriptionManager;
+class SubscriptionManagerInterface;
```

Comment:
I think it would be ok here to include `feed/SubscriptionManagerInterface.hpp` and don't make forward declaration.